fail2ban/tests/files/logs/qpsmtpd

```diff
@@ -0,0 +1,4 @@
+Thu Aug 19 23:37:09 2021 host.domain.example.com[11793]: 535 LOGIN authentication failed for marta from 2.56.59.87
```

Comment:
The test cases missing JSON record describing what must be matched by filter (once per failure line), for instance:
```
+ # failJSON: { "time": "2021-08-19T23:37:09", "match": true , "host": "192.0.2.1" }
  Thu Aug 19 23:37:09 2021 host.domain.example.com[11793]: 535 LOGIN authentication failed for marta from 192.0.2.1
```
otherwise test suite would fail in CI.